sdk/rt/MIGRATION.md

```diff
@@ -0,0 +1,153 @@
+- **Lightweight package**: Minimal dependencies for faster installation and reduced conflicts
+- **Enhanced authentication**: Dedicated `JWTAuth` class for JWT token management
+- **Streamlined configuration**: Separate `ConnectionConfig` for WebSocket settings
+
+### Breaking Changes
+
+- **Import paths**: `speechmatics.client` → `speechmatics.rt`
+- **Client class**: `WebSocketClient` → `AsyncClient`
+- **Method names**: `.run()` → `.transcribe()` for transcription operations
+- **Event registration**: `.add_event_callback()` → `.on()`
+- **Event types**: `ServerMessageType.AddTranscript` → `ServerMessageType.ADD_TRANSCRIPT`
+- **Authentication**: API key passed directly to client or through environment variable instead of `ConnectionConfig`
+- **JWT setup**: Separate `JWTAuth` class instead of `ConnectionConfig.generate_temp_token` parameter
+- **Configuration**: `ConnectionConfig` now only handles WebSocket connection settings
+- **CLI removed**: CLI will be coming back as a separate package
+
+## Installation
+
+``` bash
+pip install speechmatics-rt
+```
+
+## Basic Usage
+
+Before
+
+```python
+from speechmatics.client import WebSocketClient
+from speechmatics.models import TranscriptionConfig
+
+client = WebSocketClient(API_KEY)
+
+conf = TranscriptionConfig(language=LANGUAGE)
+
+await client.run(audio_stream, conf)
+```
+
+After
+
+```python
+from speechmatics.rt import AsyncClient, TranscriptionConfig
+
+async with AsyncClient(API_KEY) as client:
+
+    conf = TranscriptionConfig(language=LANGUAGE)
+
+    await client.run(audio_stream, transcription_config=conf)
+```
+
+## Event callbacks usage
+
+Before
+
+```python
+from speechmatics.client import WebSocketClient
+from speechmatics.models import ServerMessageType, TranscriptionConfig
+
+client = WebSocketClient(API_KEY)
+
+def on_transcript(msg: dict):
+    print(msg)
+
+client.add_event_callback(ServerMessageType.AddTranscript, on_transcript)
+
+conf = TranscriptionConfig(language=LANGUAGE)
+
+await client.run(audio_stream, conf)
+```
+
+After
+
+```python
+from speechmatics.rt import AsyncClient, TranscriptionConfig, ServerMessageType
+
+def on_transcript(msg: dict):
+    print(msg)
+
+async with AsyncClient(API_KEY) as client:
+    client.on(ServerMessageType.ADD_TRANSCRIPT, on_transcript)
+
+    conf = TranscriptionConfig(language=LANGUAGE)
+
+    await client.transcribe(audio_stream, transcription_config=conf)
+```
+
+## JWT authentication
+
+Before
+
+```python
+from speechmatics.client import WebSocketClient
+from speechmatics.models import TranscriptionConfig
+
+client = WebSocketClient(API_KEY)
+
+conf = TranscriptionConfig(language=LANGUAGE)
+
+await client.run(audio_stream, conf)
+```
+
+After
+
+```python
+from speechmatics.rt import AsyncClient, TranscriptionConfig
+
+async with AsyncClient(API_KEY) as client:
+    conf = TranscriptionConfig(language=LANGUAGE)
+    await client.run(audio_stream, transcription_config=conf)
+```
+
+## JWT authentication usage
```

Comment:
I'm slightly confused, feels like you have a duplicate section here on JWT auth?